controllers/user/UserController.php

```diff
@@ -110,10 +112,48 @@ public function userLogin($username,$password){
 
         $jwt = JWT::encode($data, $config->jwtKey, 'HS512');
 
+        //Accept the privacy policy page
+        $this->acceptPrivacy($user);
+
         return $this->returnSuccess('Success', 200, [
             'auth_token' => $jwt,
             'expired_at' => (!isset($data['exp'])) ? 0 : $data['exp']
         ]);
 
     }
+
+    /**
+     * Accept the privacy policy, After the login
```

Comment:
after registration, right? not login

Reply:
Hi @donni106 
Login is the part of registration, after the registration user is trying to login directly and this is mentioned in issue-2.
So, privacy policy code will run after the user successfully login.

Reply:
maybe thats the reason for the problems i sometimes run into. could it be possible, that after login some auto logic is triggered to check the privacy policy and show the HTML page for me instead of running the new privacy policy code before that accepts it?

Reply:
Hi @donni106 
Can you elaborate what you are saying?
do you mean that, after login show the privacy policy's HTML page before it accept?

Reply:
yes i think we mean the same. could it happen that after login the privacy policy of legal module is returned (because it is not yet accepted), before the action to accept it was called?

Reply:
Hi @donni106 
We have the few queries:
1.  you are trying to say that privacy policy's html will be returned after login but what, if we are accepting privacy policy during registration (i.e dataPrivacyCheck: true)

And for dataPrivacyCheck: false, we are showing error(As you mentioned in another resolve conversation).
![image](https://user-images.githubusercontent.com/103625795/169788421-084c7f81-ee98-468e-b97a-76cb9f9aa544.png)


So, can you tell me what screnario or case, the privacy policy's HTML will be returned.

2. API blocking : If we are not accepting privacy policy then all other endpoint will be accessible in the APIs.
3.  The login API does not return any privacy policy data, if it is not accepted so, will we create a login API in which we are showing privacy policy information if user not accepted during registration?.

Reply:
i do not understand exactly.
we do not want to allow registration without accepted privacy policy. so there do not need to be a case for that. a login for such account will not be possible, as it was not registered. with this situation there also could not be any calls to other apis, as registration failed and login is not possible.

everything should behave like this from your point of view?

Reply:
Hi @donni106 
We are making two scenarios

1. We are implementing a logic before registration where we are checking legal module is activated or not. 
and if legal module is activated then it will check 'legal' key and 'dataPrivacyCheck'. If legal key is missing or dataPrivacyCheck is false then it will throw an error and user will be not registered.

2. If legal module is not activated then user will be registered normally as before.

is it make sense?

Reply:
new code looks good i guess, but you committed with message `Allow to read public contents without login through API #6`. could you reword this please?

Reply:
Hi @donni106 
done